pkg/rancher-components/src/components/RcDropdown/RcDropdown.test.ts

```diff
@@ -96,7 +96,7 @@ describe('component: RcDropdown.vue', () => {
     await wrapper.findComponent(vDropdownMock).vm.$emit('apply-show');
     await wrapper.vm.$nextTick();
 
-    expect(dropdownTarget.style.overflow).toBe('auto');
```

Comment:
It appears that asserting that the dropdown target contains the `overflow: auto` style becomes quite tricky after moving the `overflow` style into the `dropdownTarget` class. I'm open for suggestions on how to better retain this assertion.